### DISCUSSION TITLE
CSV Export for popular Tax websites.

### DISCUSSION THREAD

**Discussion by hysticali** [pwillworth/dfkreport#2] [Ideas]
It would be nice to be able to download an edited CSV file that would work with the top Crypto Tax websites. In particular I am looking for Koinly support. See Stake.tax for an example of CSV export options.

**Comment by pwillworth** [comment]
Thanks, awesome idea, I'll check that out

**Comment by pwillworth** [comment]
Adding reference link shared by gannon in discord about the format for when I get to this https://help.koinly.io/en/articles/3662999-how-to-create-a-custom-csv-file-with-your-data

**Comment by pwillworth** [comment]
FYI - started work on this feature yesterday.  Got most of the export format functionality done, but it require some data object changes that will require database maintenance, so planning and testing that later this week.

**Comment by pwillworth** [comment]
Koinly Universal CSV export format option is now added!

**Reply by collectxyznuisance** [reply]
Getting a lot of errors like this upon import: 
![image](https://user-images.githubusercontent.com/90934269/152379428-6581bcd4-79df-4ed0-bfa6-e97b1feef998.png)

Any advice?

**Reply by pwillworth** [reply]
For the JewelToken thing, I updated the output to be Jewel instead, hopefully Koinly recognizes that, I'm sure exactly what it is expecting for token names.  That amount error is really weird.  It might be complaining about liquidity token swaps where 3 tokens are involved because the koinly import is not setup to handle swaps where 3 tokens are involved.  You might try removing those rows.

**Reply by hysticali** [reply]
I did an import into Koinly yesterday and received a few errors, but went through the transactions it imported and everything looked good. This is definitely very helpful.
